pylint/testutils/run.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+from pathlib import Path
+from typing import Any
+
+from pylint.lint import Run as LintRun
+from pylint.lint.run import UNUSED_PARAM_SENTINEL
+from pylint.reporters.base_reporter import BaseReporter
+from pylint.testutils.lint_module_test import PYLINTRC
+
+
+def _add_rcfile_pylintrc_to_args(pylintrc: str | Path, args: list[str]) -> list[str]:
+    """Add the rcfile option to the pylintrc path to an args list."""
+    if not any("--rcfile" in arg for arg in args):
+        args.insert(0, f"--rcfile={pylintrc}")
+    return args
+
+
+def add_rcfile_default_pylintrc(args: Sequence[str]) -> list[str]:
+    """Add a default pylintrc with the rcfile option in a list of pylint args."""
+    return _add_rcfile_pylintrc_to_args(PYLINTRC, list(args))
```

Comment:
Why do we need the private function?

Reply:
I supposed it could be used to add another pylintrc in args, but doing "--rcfile={pylnitrc}" directly is better. YAGNI.